doc/frameworks/djl/using_djl.rst

```diff
@@ -19,238 +16,53 @@ For general information about using the SageMaker Python SDK, see :ref:`overview
-        "my_sagemaker_role",
-        dtype="fp16",
+        model_id="<hf hub id | s3 uri>",
+        role="my_sagemaker_role",
         task="text-generation",
-        number_of_partitions=2 # number of gpus to partition the model across
     )
 
     # Deploy the model to an Amazon SageMaker Endpoint and get a Predictor
     predictor = djl_model.deploy("ml.g5.12xlarge",
                                  initial_instance_count=1)
 
-If you want to use a specific backend, then you can create an instance of the corresponding model directly.
+Alternatively, you can provide full specifications to the DJLModel to have full control over the model configuration:
 
 .. code:: python
 
-    # Create a model using the DeepSpeed backend
-    deepspeed_model = DeepSpeedModel(
-        "s3://my_bucket/my_saved_model_artifacts/", # This can also be a HuggingFace Hub model id
-        "my_sagemaker_role",
-        dtype="bf16",
-        task="text-generation",
-        tensor_parallel_degree=2, # number of gpus to partition the model across using tensor parallelism
-    )
-
-    # Create a model using the HuggingFace Accelerate backend
-
-    hf_accelerate_model = HuggingFaceAccelerateModel(
-        "s3://my_bucket/my_saved_model_artifacts/", # This can also be a HuggingFace Hub model id
-        "my_sagemaker_role",
-        dtype="fp16",
-        task="text-generation",
-        number_of_partitions=2, # number of gpus to partition the model across
-    )
-
-    # Create a model using the FasterTransformer backend
-
-    fastertransformer_model = FasterTransformerModel(
-        "s3://my_bucket/my_saved_model_artifacts/", # This can also be a HuggingFace Hub model id
-        "my_sagemaker_role",
-        data_type="fp16",
+    djl_model = DJLModel(
+        model_id="<hf hub id | s3 uri>",
+        role="my_sagemaker_role",
         task="text-generation",
-        tensor_parallel_degree=2, # number of gpus to partition the model across
+        engine="Python",
+        env={
+            "OPTION_ROLLING_BATCH": "lmi-dist",
+            "TENSOR_PARALLEL_DEGREE": "2",
+            "OPTION_DTYPE": "bf16",
+            "OPTION_MAX_ROLLING_BATCH_SIZE": "64",
+        },
+        image_uri=<djl lmi image uri>,
     )
 
-    # Deploy the model to an Amazon SageMaker Endpoint and get a Predictor
-    deepspeed_predictor = deepspeed_model.deploy("ml.g5.12xlarge",
-                                                 initial_instance_count=1)
-    hf_accelerate_predictor = hf_accelerate_model.deploy("ml.g5.12xlarge",
-                                                         initial_instance_count=1)
-    fastertransformer_predictor = fastertransformer_model.deploy("ml.g5.12xlarge",
-                                                                 initial_instance_count=1)
-
-Regardless of which way you choose to create your model, a ``Predictor`` object is returned. You can use this ``Predictor``
```

Comment:
should we still leave in some docs explaining that a `Predictor` is returned on `deploy()`?

Reply:
yea i can do that, I think i deleted this line by mistake!